website/docs/d/secretsmanager_secret_version.html.markdown

```diff
@@ -42,4 +42,5 @@ data "aws_secretsmanager_secret_version" "by-version-stage" {
 * `arn` - The ARN of the secret.
 * `id` - The unique identifier of this version of the secret.
 * `secret_string` - The decrypted part of the protected secret information that was originally provided as a string.
+* `secret_binary` - The decrypted part of the protected secret information that was originally provided as a binary.
```

Comment:
We should note this is base64 encoded. I'll make the very minor edit post-merge. 😄